integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalRmiTunneling.java

```diff
@@ -465,6 +599,68 @@ public void testExternalRmiAccessThruHttpsTunneling() {
         .until(runJmsClient(new String(javasCmd)));
   }
 
+  // Run the RMI client inside K8s Cluster
+  private void runPodClient(int serverCount, boolean checkConnection) {
```

Comment:
change method name to runClientInsidePod

Reply:
modified the method name